libnmstate: do not report perm_mac_address when showing

ffmancera opened the pull request:
Nmstate is currently showing _perm_mac_address when showing. This is not
correct because perm_mac_address is only for metadata.

Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

cathay4t reviewed (changes requested):
I tested in my VM. The `_permanent_mac_address` still exists in `libnmstate.show()`.

ffmancera committed:
libnmstate: do not report perm_mac_address when showing

Nmstate is currently showing _perm_mac_address when showing. This is not
correct because perm_mac_address is only for metadata. The metadata must
be removed when reporting the state and when verifying but it needs to
stay when modifying the state.

Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>
